packages/components/tsconfig.json

```diff
@@ -1,25 +1,26 @@
 {
   "compilerOptions": {
-    "outDir": "dist/",
-    "module": "commonjs",
-    "target": "es5",
-    "lib": ["es5", "es6", "dom"],
-    "sourceMap": true,
     "allowJs": false,
-    "jsx": "react",
-    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
     "baseUrl": "src",
+    "declaration": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
     "forceConsistentCasingInFileNames": true,
+    "jsx": "react",
+    "lib": ["es5", "es6", "dom"],
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noImplicitAny": false,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    "noImplicitAny": false,
+    "noUnusedLocals": true,
+    "outDir": "dist/",
+    "sourceMap": true,
+    "skipLibCheck": true,
```

Comment:
alphabetized + added this line to actually skip `node_modules` (https://github.com/Microsoft/TypeScript/issues/14293)

Reply:
Nice!